kytos/cli/commands/controller/parser.py

```diff
@@ -0,0 +1,43 @@
+"""kytos - The kytos command line.
+
+You are at the "controller" command.
+
+Usage:
+       kytos controller start
+       kytos controller stop
+       kytos controller restart
+       kytos controller -h | --help
+
+Options:
+
+  -h, --help    Show this screen.
+
+Common napps subcommands:
+
+  start        Run kytos controller in daemon context.
+  stop         Stop kytos controller in daemon context.
```

Comment:
`stop` works only if kytos is running in daemon context? What if kytos is running in foreground?